tools/ports/freetype.py

```diff
@@ -488,9 +488,9 @@ def show():
 #if ( __GNUC__ >= 2                         || \
       defined( __IBM__TYPEOF__ )            || \
       ( __SUNPRO_C >= 0x5110 && !__STDC__ ) )
-#define TYPEOF( type )  (__typeof__ (type))
+#define FT_TYPEOF( type )  (__typeof__ (type))
 #else
-#define TYPEOF( type )  /* empty */
+#define FT_TYPEOF( type )  /* empty */
```

Comment:
This changes the definition - do the uses not need to be updated too? I'm just confused how this works.

Reply:
Yes maybe you are right. I assumed this file was just the shipped header, but it is also used during the build.

Given that you own the repo, a much easier solution is to cherry-pick https://github.com/emscripten-ports/FreeType/commit/5931268eec into the git tag `version_1` that we pull in here. I am not why you ended up archiving that repo, but just getting this one commit in would be a big improvement for us (and harmless for others).

Reply:
Hmm, I'm not sure why that was archived. @sbc100 did we plan to use upstream perhaps?

Reply:
I think we should look into upgrading our version of freetype.   I'll take a look.

Reply:
Should be done in #22585